device/ingrasys/x86_64-ingrasys_s9280_64x-r0/platform.asic

```diff
@@ -0,0 +1 @@
+barefoot
```

Comment:
Typo in the filename?

Reply:
Great thanks! I will remove the extra file